src/Microsoft.NetCore.Analyzers/UnitTests/Tasks/DoNotCallBeginInvokeOnDelegateTests.cs

```diff
@@ -0,0 +1,656 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+
+using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
```

Comment:
❕ These are not security analyzers

```suggestion
using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Reply:
📝 Previously, `CSharpSecurityCodeFixVerifier` was used due to a limitation in the test library. This limitation no longer applies and `CSharpCodeFixVerifier` will now work without the need for other workarounds.